code/mob/living/critter.dm

```diff
@@ -1033,8 +1034,16 @@ ADMIN_INTERACT_PROCS(/mob/living/critter, proc/modify_health)
 
 	update_inhands()
 		var/handcount = 0
-		for (var/datum/handHolder/HH in hands)
+		for (var/datum/handHolder/HH as anything in src.hands)
 			handcount++
+			if (HH.object_for_inhand)
+				var/obj/item/I = new HH.object_for_inhand
```

Comment:
does this have to instantiate or can it use `initial`?

Reply:
seems like its always null on initial